src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -131,6 +156,12 @@ export const validationSchema_v2 = Yup.object()
   })
   .required();
 
+export const newVenueSchema = Yup.object()
+  .shape<NewVenueSchema>({
+    name: venueNameValidation,
+  })
+  .required();
```

Comment:
Probably worth also including the 'only this data and nothing else is allowed' property on this schema as well.

Reply:
`noUnknown` (like what is being done in `src/types/sounds.ts`):

- https://github.com/jquense/yup#objectnounknownonlyknownkeys-boolean--true-message-string--function-schema